ui/dashboards/src/components/Panel/Panel.test.tsx

```diff
@@ -60,40 +115,53 @@ describe('Panel', () => {
     definition ??= createTestPanel();
 
     renderWithContext(
-      <TimeRangeProvider timeRange={{ pastDuration: '1h' }}>
-        <VariableProvider
-          initialVariableDefinitions={[
-            {
-              kind: 'TextVariable',
-              spec: {
-                name: 'foo',
-                value: 'bar ',
+      <ThemeProvider theme={testTheme}>
+        <TimeRangeProvider timeRange={{ pastDuration: '1h' }}>
+          <VariableProvider
+            initialVariableDefinitions={[
+              {
+                kind: 'TextVariable',
+                spec: {
+                  name: 'foo',
+                  value: 'bar ',
+                },
               },
-            },
-          ]}
-        >
-          <DataQueriesProvider definitions={[]}>
-            <Panel definition={definition} editHandlers={editHandlers} panelOptions={panelOptions} />
-          </DataQueriesProvider>
-        </VariableProvider>
-      </TimeRangeProvider>
+            ]}
+          >
+            <DataQueriesProvider definitions={[]}>
+              <Panel definition={definition} editHandlers={editHandlers} panelOptions={panelOptions} />
+            </DataQueriesProvider>
+          </VariableProvider>
+        </TimeRangeProvider>
+      </ThemeProvider>
     );
   };
 
-  // Helper to get the panel once rendered
   const getPanel = (): HTMLElement => screen.getByTestId('panel');
 
+  it('should render plugin actions in header', async () => {
+    renderPanel();
+    const panel = getPanel();
+    userEvent.hover(panel);
+
+    await waitFor(() => {
+      const exportButton = screen.getAllByLabelText('Export CSV');
+      expect(exportButton.length).toBeGreaterThan(0);
+      exportButton.forEach((button) => {
+        expect(button).toHaveTextContent('Export');
+      });
+    });
```

Comment:
Maybe this could be simplified to:
```suggestion
    const exportButton = screen.getByLabelText('Export CSV');
    expect(exportButton).toBeInTheDocument();
```